modules/update-availability.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: CONCEPT
+[id="update-availability_{context}"]
+= Common questions about update availability
+
+There are several factors that affect if and when an update is made available to an {product-title} cluster. The following list provides common questions regarding the availability of an update:
+
+*What are the differences between each of the different update channels?*
+
+* A new release is initially added to the `candidate` channel and is often available as an update target for clusters on the `candidate` channel.
+
+* After successful internal and external testing, a release on the `candidate` channel is supported by being promoted to the `fast` channel and is often available to clusters on the `fast` channel.
+
+* After a short delay and more successful testing, a release on the `fast` channel is finally promoted to the `stable` channel and is often available to clusters on the `stable` channel. This delay represents the only difference between the `fast` and `stable` channels.
+
+* Releases promoted to the `stable` channel are simultaneously promoted to the `eus` channel.
+The primary purpose of the `eus` channel is to serve as a convenience for clusters performing an EUS-to-EUS update.
+
+*Is a release on the `stable` channel safer or more supported than a release on the fast channel?*
+
+* If a regression is identified for a release on a `fast` channel, it will be resolved and managed to the same extent as if that regression was identified for a release on the `stable` channel.
+
+* The only difference between releases on the `fast` and `stable` channels is that a release only appears on the `stable` channel after it has been on the `fast` channel for a short period and has not had any major issues.
+
+** This delay is used to further test and gather data about a release that is on the `fast` channel, but otherwise there is no difference between releases on each of the channels.
+
+*What does it mean if an update is supported but not recommended?*
+
+* Red Hat continuously evaluates data from multiple sources to determine whether updates from one version to another lead to issues.
+If an issue is identified, an update path may no longer be recommended to users.
```

Comment:
nit: possibly "to users" -> "to some clusters" to leave clear space for it continuing to be recommended for other clusters and generally being per-cluster advice?